README.md

```diff
@@ -97,6 +97,14 @@ app.UseOpenIdConnectServer(options => {
 });
 ```
 
+Also, remember to add authentication services:
```

Comment:
How about: `In order for the OpenID Connect server middleware to work properly, the authentication services must be registered in the DI container`?